packages/flutter_tools/lib/src/artifacts.dart

```diff
@@ -211,6 +215,17 @@ class ArtifactStore {
     return _engineRevision;
   }
 
+  static String _expectedEngineRevision;
+
+  static String get expectedEngineRevision {
+    if (_expectedEngineRevision == null) {
+      File revisionFile = new File(path.join(flutterRoot, 'packages', 'flutter', 'packages', 'sky_engine', 'REVISION'));
```

Comment:
We should use `flutter_tools` here rather than `flutter`.  There's no guarantee that we've run `pub get` on the flutter package for end-developers, but we must have run `pub get` for flutter_tools in order to be executing this code.

I guess that would require adding a dependency on `package:flutter` from `flutter_tools`, which doesn't make much sense.  Maybe we should teach the `bin/flutter` script to `pub get` package:flutter as well....